squad/frontend/templates/squad/index.jinja2

```diff
@@ -27,4 +27,10 @@
 </div>
 
 
+{% if user.is_authenticated %}
```

Comment:
Maybe this block should be moved to the top? The button is a bit 'buried' in the page.
Should we allow creating new groups to all authenticated users? This seems a bit dangerous to me.